### PR TITLE
Added new forms for bug, feature and question

### PR DESCRIPTION
**Pull request by nvvfedorov**
Added new Github forms:

* Bug Report: [preview](https://github.com/NVIDIA/dcgm-exporter/blob/new-issue-forms/.github/ISSUE_TEMPLATE/bug_report_form.yml)

*  Feature Request: [preview](https://github.com/NVIDIA/dcgm-exporter/blob/new-issue-forms/.github/ISSUE_TEMPLATE/feature_request_form.yml)

* Question: [preview](https://github.com/NVIDIA/dcgm-exporter/blob/new-issue-forms/.github/ISSUE_TEMPLATE/submit-question.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        - Must have.
+        - Should have.
+        - Nice to have.
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Please provide a clear description of problem this feature solves
```
Comment (glowkey) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:51]: "of the problem"

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      description: Real usage examples are especially helpful, non-code.
+    validations:
+      required: true
+
+  - type: textarea
+    id: Feature_Description
+    attributes:
+      label: Feature Description
+      description: Please provide clear description of the feature you request (refer to [User Story format](https://www.atlassian.com/agile/project-management/user-stories#:~:text=User%20story%20template%20and%20examples) and [EARS format](https://ieeexplore.ieee.org/document/5328509))
```
Comment (glowkey) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:60]: "Provide a clear description of the requested feature"

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      placeholder: >
+        For new feature request, please use one of the following format to describe the feature
```
Comment (glowkey) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:62]: "requests" "formats"

**Review by glowkey — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this feature request
+      options:
+        - Must have.
+        - Should have.
+        - Nice to have.
+    validations:
+      required: true
```
Comment (rohit-arora-dev) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:46]: I believe most will go with "Must have". The decision to support something and criticality should be a team decision.

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      description: Is DCGM-Exporter running on bare metal or in a virtual environment, container, pod, etc?
+  - type: textarea
+    id: deployment
+    attributes:
+      label: How did you deploy the dcgm-exporter and what is the configuration?
+      description: Tell us how you deployed DCGM-Exporter. Did you use helm, build from source or use the GPU Operator?
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: How can we reproduce the issue?
```
Comment (rohit-arora-dev) [inline .github/ISSUE_TEMPLATE/bug_report_form.yml:63]: nit:
How to reproduce the issue?

File: .github/ISSUE_TEMPLATE/submit-question.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this question!
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the question.
```
Comment (rohit-arora-dev) [inline .github/ISSUE_TEMPLATE/submit-question.yml:29]: May be:
"Ask your question" or same as description "What is your question?"

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 1 above)
Comment (nvvfedorov) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml]: I made the change.

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 3 above)
Comment (nvvfedorov) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml]: I made the change.

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 4 above)
Comment (nvvfedorov) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml]: I removed the section to let us decide on a priority.

File: .github/ISSUE_TEMPLATE/submit-question.yml (hunk 6 above)
Comment (nvvfedorov) [reply .github/ISSUE_TEMPLATE/submit-question.yml]: I made the change.

**Review by rohit-arora-dev — APPROVED** [review]
Thank you @nvvfedorov this will make process much easier and communication faster.

/LGTM

**Review by glowkey — APPROVED** [review]

**Pull request merged**